cmd/ddev/cmd/config_test.go

```diff
@@ -514,6 +514,8 @@ func TestConfigDatabaseVersion(t *testing.T) {
 	// add a database into the config and nothing else
 	for _, dbTypeVersion := range versionsToTest {
 		_ = app.Stop(true, false)
+		_, err := exec.RunHostCommand(DdevBin, "delete", "-Oy", t.Name())
+		assert.NoError(err)
```

Comment:
I've been leaning more toward require() lately, which stops the test. Less time chasing irrelevant things. It's not necessary to accept this, I'm just commenting.
```suggestion
		require.NoError(t, err)
```

Reply:
Might be worth making a push to change all of them at some stage and get CI to catch that. 
I'll change this if you want me to use the `omitempty` discussed in all the other comments - but if you're happy leaving that as is then I'll leave this as well.

Reply:
It's not important. You're just such an important contributor that I'm doing a bit of coaching. Nothing wrong with what's here.